macos/FluentUITestViewControllers/TestAvatarViewController.swift

```diff
@@ -19,6 +19,7 @@ private struct TestData {
 	static let maor = TestIdentity(name: "Maor Sharett", email: "Maor.Sharett@example.com", image: nil)
 	static let annieBoyl = TestIdentity(name: "Annie Boyl Lind", email: "annie.boyl@example.com", image: nil)
 	static let kat = TestIdentity(name: nil, email: "Kat.Larrson@example.com", image: nil)
+	static let turkey = TestIdentity(name: "🦃", email: "🦃🦃🦃🦃🦃🦃🦃🦃@example.com", image: nil)
```

Comment:
Can we add this nice bird to our iOS demo?

Reply:
Also, seems a bit odd that the first letter we find is the e from @example.com. Are we sure this algorithm isn't biased against certain languages?

Reply:
I'd suggest adding a few more test cases - different languages, different names, different combinations of emojis and characters.

Reply:
Two thoughts:
* Our iOS demo currently doesn't have much coverage at all for name variety -- it's architected to mainly pivot on appearance style. Adding various names as a test case would potentially require a substantial rethinking of the control, which I'm not currently budgeted to undertake. But I'll make sure it gets on our backlog, with this particular test case called out.
* I agree that it's odd that we skip all the way to "e", but the Avatar is currently behaving as specced -- to only use Latin alphabet letters as initials. This aligns with all Fluent platforms, and reconsideration needs to be a cross-platform feature request.

Reply:
1. I think you'd be fine just adding a new entry to the list of sample personas from the persona list view demo controller?
2. That's fair, and something we should bring up with the other platforms, probably.

Reply:
Fun fact: the iOS algorithm does behave differently here than macOS or web -- it only ever checks the first letter of a string for unicode-ness, as opposed to the other platforms that check iteratively through the whole string.

I'm gonna go ahead and _not_ change that now because this PR is susceptible to a whole lot of bloat, but we should probably revisit that at some point.